components/web/php/php-8_2-ext-redis/Makefile

```diff
@@ -20,6 +20,7 @@ include ../../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		redis
 COMPONENT_VERSION=	6.0.2
+COMPONENT_REVISION=	1
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
```

Comment:
You don't need to set COMPONENT_REVISION because we haven't published this package before. The first try failed and so no package made it into our repository yet.